src/main/target/common_pre.h

```diff
@@ -493,3 +492,66 @@
 #if defined(USE_POSITION_HOLD) && !defined(USE_GPS)
 #error "USE_POSITION_HOLD requires USE_GPS to be defined"
 #endif
+
+// backwards compatibility for older config.h targets
+#ifndef GYRO_CONFIG_USE_GYRO_1
```

Comment:
IMO only one version should be used - either gyro.h or common_pre.h. 
Or, if really necessary, it can be moved to separate header file.

Reply:
They are defined in two places now... Only has to be defined once in either gyro.h or common_pre.h